### DISCUSSION TITLE
When will WeChat be added?

### DISCUSSION THREAD

**Discussion by U0hZiY** [JulianPrieber/littlelink-custom#803] [Feature requests]
Is it possible to add a WeChat adding method?

**Comment by JulianPrieber** [comment]
I'm not familiar with WeChat. Do you share it as a link or with like a phone number?

**Reply by U0hZiY** [reply]
I have used similar websites. They allow users to upload WeChat QR code images, then click to open, scan the code to add, or long press the screen. WeChat does not have a sharing link. Users can only add friends by searching by phone number. At least in China.
I like this project very much, and I am very happy to get a reply,thank you bro.

**Reply by JulianPrieber** [reply]
We are introducing a feature that would make implementing something like this possible.
I'll put it on our list, but in the meantime maybe someone else wants to implement it.

**Reply by U0hZiY** [reply]
> I'm not familiar with WeChat. Do you share it as a link or with like a phone number?
> 
> 

There are only two methods: searching for phone numbers and scanning QR codes. We often use the QR codes, so i think linkstack my need the function of uploading pictures.
